types/n3/index.d.ts

```diff
@@ -113,6 +113,7 @@ export type ParseCallback = (error: Error, quad: Quad, prefixes: Prefixes) => vo
 
 export interface N3Parser {
     parse(input: string, callback: ParseCallback): void;
+    parse(input: string): Quad[];
```

Comment:
Please change the order of signatures for parse.

Reply:
thanks, done